web/html/xui/form/tls_settings.html

```diff
@@ -112,7 +112,7 @@
                 <template slot="title">
                     <span>{{ i18n "pages.client.renew" }}</span>
                 </template>
-                Short IDs
+                Short ID
```

Comment:
In outbounds we should use one. But here we should define all shortIds. Please don't remove `s` here